common/table.js

```diff
@@ -1958,16 +1971,48 @@
                         };
                         ErrorService.handleException(res.issues, false, false, cb, cb);
                     } else {
-                        if ($rootScope.savedQuery && $rootScope.savedQuery.rid) {
+                        if ($rootScope.savedQuery && $rootScope.savedQuery.showUI && !$rootScope.savedQuery.updated) {
+                            // to prevent the following code and request from triggering more than once
+                            // NOTE: doesn't matter if the update is successful or not, we are only preventing this block from triggering more than once
+                            $rootScope.savedQuery.updated = true;
+
                             var rows = [{}],
                                 updateRow = rows[0];
 
                             updateRow.RID = $rootScope.savedQuery.rid
                             updateRow.last_execution_time = "now";
 
+                            // create this fake table object so getStackNode works
+                            var fauxTable = {
+                                name: $rootScope.savedQuery.mapping.table,
+                                schema: {
+                                    name: $rootScope.savedQuery.mapping.schema
+                                }
+                            }
+
+                            var stackPath = logService.getStackPath(logService.logStackPaths.SET, logService.logStackPaths.SAVED_QUERY_CREATE_POPUP);
+                            var currStackNode = logService.getStackNode(logService.logStackTypes.SAVED_QUERY, fauxTable);
+
+                            var logObj = {
+                                action: logService.getActionString(logService.logActions.UPDATE, stackPath),
+                                stack: logService.addExtraInfoToStack(logService.getStackObject(currStackNode), {
+                                    "num_updated": 1,
+                                    "updated_keys": {
+                                        "cols": ["RID"],
+                                        "vals": [$rootScope.savedQuery.rid]
```

Comment:
This should be `[[$rootScope.savedQuery.rid]]`. The outer array is for the rows that we're updating, and the inner one is for the values that correspond do the `cols` array.